### ISSUE TITLE
Guide Request: With Prisma

### ISSUE THREAD

**Issue by ile** [solidjs/solid-docs#736]
### 📚 Subject area/topic

SolidStart

### 📋 Page(s) affected (or suggested, for new content)

https://docs.solidjs.com/solid-start

### 📋 Description of content that is out-of-date or incorrect

How is the database (Prisma, sqlite) supposed to work in a production environment - after doing `npm run build` and then `npm start`? 

Where is the database file supposed to go and how can I create an empty database?

The development database file is in `./prisma/dev.db`

**Comment by thomasbuilds** [comment]
@LadyBluenotes docs don't showcase specific examples like that right?
We rather have examples in [start repo](https://github.com/solidjs/solid-start/tree/main/examples)
Issue should get closed?

**Comment by LadyBluenotes** [comment]
Yeah, I'd say if you wanna keep the examples there (and we don't need to address this with a how-to) we can close it. Do you mind opening a new one in the start repo and tagging this to follow it up? @thomasbuilds

**Comment by thomasbuilds** [comment]
> Yeah, I'd say if you wanna keep the examples there (and we don't need to address this with a how-to) we can close it. Do you mind opening a new one in the start repo and tagging this to follow it up? [@thomasbuilds](https://github.com/thomasbuilds)

There is already a [with-prisma](https://github.com/solidjs/solid-start/tree/main/examples/with-prisma) example actually so all good!
